cordapp/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/ReadmeExampleTest.kt

```diff
@@ -156,9 +163,20 @@ class ReadmeExampleTest {
                         listOf(legalAgePredicate),
                         aliceX500)).resultFuture.get()
 
-// If the verification succeeds, the store can be sure that Alice's age is above 18.
+        logger.info("If the verification succeeds, the store can be sure that Alice's age is above 18.")
 
-        println("You can buy drinks: $verified")
+        logger.info("You can buy drinks: $verified")
     }
 
+    private fun <T> StartedNodeServices.startFlow(logic: FlowLogic<T>): FlowStateMachine<T> {
+        val machine = startFlow(logic, newContext()).getOrThrow()
+
+        return object : FlowStateMachine<T> by machine {
+            override val resultFuture: CordaFuture<T>
+                get() {
+                    net.runNetwork()
+                    return machine.resultFuture
+                }
+        }
+    }
```

Comment:
Could we create separated file with the repeatable code (startFlow) from both tests?